dashboard/test/ui/features/learning_platform/level_types/free_response_contained_levels.feature

```diff
@@ -88,3 +88,21 @@ Scenario: Authorized Teacher on App Lab with free response contained level
   Then I press "runButton"
   And I see no difference for "level run"
   Then I close my eyes
+
+Scenario: Teacher can reset progress on free response contained level
+  When I open my eyes to test "reset free response contained level"
+  Given I sign in as "Teacher_Lillian"
+  And I am on "http://studio.code.org/s/allthethings/lessons/41/levels/3?enableExperiments=instructorPredictLevelReset"
+  And I rotate to landscape
+  And I wait for the page to fully load
+  And I press keys "Here is my response!" for element ".response"
+  Then I press "runButton"
+  Then I press "resetButton"
+  Then I click selector "button:contains('Delete Answer')"
```

Comment:
Is there a way you could show here that the .response element gets cleared out? Or maybe I'm just not as familiar with how eyes tests work in comparison to other .feature tests. Like what does "And I see no difference..." mean when it's looking at a new label each time?

Reply:
honestly, I think I might want to remove the eyes test portion of this (it's only one checkpoint anyway) and just do what you're suggesting (assuming I can). Checking that .response is cleared out sounds like a good thing to do either way

Reply:
hm actually I'm not sure how to verify that a textbox is empty. I can verify that a textbox contains other text but as I want it to contain nothing, I'm not sure how to do it, so I'm tempted to leave this as is.

To answer your other question, the label on "And I see no difference" allows applitools to track the checkpoints across test runs so that it knows when something has changed

Reply:
thanks I will try that!